### PR TITLE
User-friendlization of the OVAL lab

### PR DESCRIPTION
**Pull request by matejak**
Added more explicit instructions + moved some content to note/tip boxes.

### CODE REVIEW

**Comment by dahaic** [comment]
There is numbering issue at "The OVAL test repeats itself a little bi" string :)

**Comment by dahaic** [comment]
Anatomy of an existing check-remediation pair: Opening of file browser is already in "Before you start" section

**Comment by dahaic** [comment]
"When dealing with the rule check, we will focus on those two aspects" I would rephrase  to something like "When dealing with the rule check, there are additional aspects to have in mind:"

**Comment by dahaic** [comment]
5.3.1 Bash remediation: start with short explanation why we are beginning with Bash. "We will start with the more straightforward part of the combination - Bash remediation"

**Comment by dahaic** [comment]
"Either way, the remediation body looks like this:" Unfortunately the source is slightly different. Because the HTML cannot be used for OVAL, maybe remove this option, and just explain how to get to to the source? And mention that the source has header that is processed by build system.

**Comment by dahaic** [comment]
piece-wide -> piece by piece? Part by part?

**Comment by dahaic** [comment]
I still have to go through `Lab 5.3.2 OVAL check` and more...

**Comment by matejak** [comment]
LGTM
